jobs/build-fcos-buildroot.Jenkinsfile

```diff
@@ -0,0 +1,161 @@
+def pipeutils, streams, official
+def gitref, commit, shortcommit
+node {
+    checkout scm
+    pipeutils = load("utils.groovy")
+    streams = load("streams.groovy")
+    official = pipeutils.isOfficial()
+}
+
+properties([
+    pipelineTriggers([
+        githubPush()
+    ]),
+    parameters([
+      string(name: 'ARCHES',
+             description: 'Space-separated list of target architectures',
+             defaultValue: "x86_64",
+             trim: true),
+      string(name: 'CONFIG_GIT_URL',
+             description: 'Override the fedora-coreos-config git repo to use',
+             defaultValue: "https://github.com/coreos/fedora-coreos-config.git",
+             trim: true),
+      string(name: 'CONFIG_GIT_REF',
+             description: 'Override the fedora-coreos-config git ref to use',
+             defaultValue: "testing-devel",
+             trim: true),
+      string(name: 'CONTAINER_REGISTRY_REPO',
+             description: 'Override the registry to push the container to',
+             defaultValue: "quay.io/coreos-assembler/fcos-buildroot",
```

Comment:
It seems to me that if we did at the top e.g.

```
def container_name = "fcos-buildroot";
```

and then used it here, we could more clearly highlight things specific to this contaienr build.